src/components/Editor/components/Main.js

```diff
@@ -71,19 +72,15 @@ class Main extends React.Component {
   renderOpenPanelButton = () => {
     const { panelVisible, handleOnVisibleChange } = this.props;
 
-    //if the left panel is closed, show an empty div
-    if (panelVisible) {
-      return <div className="editor-expand-panel-arrow" />;
-    }
-
-    // otherwise show a > that when clicked, opens the panel
+    //if the left panel is closed, show nothing
+    //otherwise show hamburger icon
     return (
       <div
         className="editor-expand-panel-arrow"
         title="Open Profile Panel"
         onClick={handleOnVisibleChange}
       >
-        >
+        {panelVisible ? "" : <FontAwesomeIcon icon="bars" />}
```

Comment:
I think that this icon should be white - it seems inconsistent with the current colour that it is.